static/truewiki/truewiki.css

```diff
@@ -586,3 +586,177 @@ header {
     font-size: 18px;
     font-weight: bold;
 }
+
+/* Pygments - Light Mode */
```

Comment:
Please mark the source of these lines; can be something like: taken from pygments theme XXX